news/this-week-in-fluvio-0050.md

```diff
@@ -66,15 +66,15 @@ We are also exploring a developer user experience for our cloud platform. More u
 [Using DuckDB with Fluvio]
 
 
-[the docs to CDK]: ../docs/fluvio/cli/cdk
-[Using DuckDB with Fluvio]: https://infinyon.com/blog/2023/02/duckdb-fluvio/
 
 ---
 
 Get in touch with us on [Github Discussions] or join [our Discord channel] and come say hello! Watch videos on our [InfinyOn Youtube Channel]
 
 For the full list of changes this week, be sure to check out [our CHANGELOG].
 
+[cdk]: connectors/cdk.mdx
```

Comment:
It should be `[the docs to CDK` instead of `[cdk]`

Reply:
I'll pick this up in a followup

Reply:
fixed